lib/elixir/lib/supervisor.ex

```diff
@@ -194,9 +194,9 @@ defmodule Supervisor do
   For a more complete understanding of the exit reasons and their
   impact, see the "Exit reasons and restarts" section.
 
-  ## child_spec/1
+  ## child_spec/1 function
```

Comment:
While we're at it, let's add some backticks too 😉 

```suggestion
  ## `child_spec/1` function
```